sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -392,7 +417,14 @@ public DigitalTwinsServiceVersion getServiceVersion() {
     <T> Mono<DigitalTwinsResponse<T>> createOrReplaceRelationshipWithResponse(String digitalTwinId, String relationshipId, T relationship, Class<T> clazz, CreateOrReplaceRelationshipOptions options, Context context) {
         return protocolLayer
             .getDigitalTwins()
-            .addRelationshipWithResponseAsync(digitalTwinId, relationshipId, relationship, OptionsConverter.toProtocolLayerOptions(options), context)
+            .addRelationshipWithResponseAsync(
+                digitalTwinId,
+                relationshipId,
+                relationship,
+                OptionsConverter.toProtocolLayerOptions(options),
+                context.addData(AZ_TRACING_NAMESPACE_KEY, DIGITAL_TWINS_TRACING_NAMESPACE_VALUE)
+                    .addData(DIGITAL_TWIN_ID_TRACING_KEY, digitalTwinId)
+                    .addData(RELATIONSHIP_ID_TRACING_KEY, relationshipId))
```

Comment:
Where are these keys read? `AZ_TRACING_NAMESPACE_KEY` is a known azure-core key that is read by the `OpenTelemetryHttpPolicy`.  

Did you mean to add `digitalTwinId` and `relationshipId` as tracing span attributes?

cc: @samvaity

Reply:
Yes, as @srnagar said,  `DIGITAL_TWIN_ID_TRACING_KEY` and `RELATIONSHIP_ID_TRACING_KEY` are not identified keys on the tracing end(azure-core-tracing-opetelemetry). 

And hence, these attributes would not be attached to the tracing span until it's recognized and supported in the `azure-core-tracing-opetelemetry` library itself

Reply:
I see, I will remove all additional keys since they are not supported.